### PR TITLE
Add email whitelist

### PR DESCRIPTION
**Pull request by AleLudovici**
This PR adds a whitelist of user emails that are authorised to login. If unauthorised then redirect the user to an error page.

The PR also pulls in a new version of the pan domain auth library that more sensibly expires users permissions.

### CODE REVIEW

File: app/controllers/Login.scala [hunk 1]
```diff
@@ -13,4 +13,8 @@ object Login extends Controller with PanDomainAuthActions {
   def logout = Action.async { implicit request =>
     Future(processLogout)
   }
+
+  def authError(message: String) = Action.async { implicit request =>
+    ???
```
Comment (theefer) [inline app/controllers/Login.scala]: Did you mean to leave this in?

Comment (theefer) [inline app/controllers/Login.scala]: Ah sorry missed the second line of your description! :no_mouth:

File: conf/routes [hunk 2]
```diff
@@ -3,27 +3,28 @@
 # ~~~~
 
 # Content finder
-GET     /                                     controllers.Application.index
-POST    /find                                 controllers.Application.find
+GET            /                                           controllers.Application.index
```
Comment (rrees) [inline conf/routes]: Looks like the spacing has gone awry, maybe reformat this with standard Intellij settings? Or what about that magic editor file?

Comment (sihil) [inline conf/routes]: I think this is now the standard intellij settings.

**Comment by sihil** [comment]
This is ready to be reviewed now. If someone happens to see this over the weekend, I'd love to get it out before Monday.

File: conf/routes (hunk 2 above)
Comment (sihil) [inline conf/routes]: You can append `?w=0` to the end of a diff URL in github to ignore whitespace.

**Comment by rrees** [comment]
:+1: :shipit:

File: conf/routes (hunk 2 above)
Comment (rrees) [inline conf/routes]: I just don't see the point of being inconsistent if you've attempt vertical alignment

Comment (sihil) [inline conf/routes]: Not sure what you mean. IntelliJ is very opinionated about the formatting in this file. It's futile to ensure everyone has anything other than the default settings.

Comment (rrees) [inline conf/routes]: The formatting was looking unaligned in Github but checking out the code locally it looks fine

Comment (sihil) [inline conf/routes]: :+1:

**Pull request merged**